README.md

```diff
@@ -61,7 +61,7 @@ fn update_damage(
  - [`ldtk`](examples/ldtk/ldtk.rs) - An example of loading and rendering of a LDTK map. Use: `cargo run --example ldtk`. We recommend checking out: [`bevy_ecs_ldtk`](https://crates.io/crates/bevy_ecs_ldtk).
  - [`random_map`](examples/random_map.rs) - A bench of editing all of the tiles every 100 ms.
  - [`remove_tiles`](examples/remove_tiles.rs) - An example showing how you can remove tiles by using map_query
- - [`tiled_rotate`](examples/tiled_rotate.rs) - An example of loading and rendering of a tiled map editor map with flipping and rotation. Use: `cargo run --example tiled_rotate`
+ - [`tiled_rotated`](examples/tiled_rotate.rs) - An example of loading and rendering of a tiled map editor map with flipping and rotation. Use: `cargo run --example tiled_rotated`
```

Comment:
the URL here is still wrong. :)

Reply:
Thanks for the correction. I appreciate the communication.